src/goCover.ts

```diff
@@ -305,3 +309,19 @@ export function toggleCoverageCurrentPackage() {
 		});
 	});
 }
+
+export function isPartOfComment(e: vscode.TextDocumentChangeEvent): boolean {
+	let result = false
+	e.contentChanges.every(function(change, index){
+		let line = e.document.lineAt(change.range.start)
+		let text = e.document.getText(line.rangeIncludingLineBreak)
+		// Exit the loop as soon as one of the changes is not part of a comment.
+		if (!text.startsWith('//')){
+			result = false
```

Comment:
We dont need to use the `result` variable here. Instead, we can do the below

```
return e.contentChanges.every(change => {
// Just return true or false based on your logic here
}
```
The `.every` on an array stops looping over the array items the moment one of them returns `false` in the callback

Reply:
Fixed. Will update the PR once I have fixed the block comment issue.